runner/runner.go

```diff
@@ -367,7 +367,7 @@ func (r *Runner) RunEnumeration() {
 			if resp.err != nil {
 				gologger.Debug().Msgf("Failure '%s': %s\n", resp.URL, resp.err)
 			}
-			if resp.str == "" {
+			if resp.str == "" && !resp.Failed{
```

Comment:
As per the last update also failed responses should be printed out, so here it should be:
```
if resp.str == "" {
```